.travis.yml

```diff
@@ -115,6 +115,14 @@ install:
   - python setup.py build_ext --inplace
   - python setup.py develop
 
+before_script:
+  - pip install flake8
+  - echo -n "flake8 testing of ${URL} on " ; python -V
+  # stop the build if there are Python syntax errors or undefined names
+  - flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+  # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
+  - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
How strong is your preference for 127 over 80?

Reply:
Default is 79...  I ususally bang my code through Black which settles on 88 characters.  I usually bump it up to the width of the GitHub editor when tesing a new repo to not clog up the error reports.  We can set it to max-line-length=80 if that is the standard for statsmodels.